cognite/client/data_classes/workflows.py

```diff
@@ -352,6 +372,7 @@ def __init__(
         description: str | None = None,
         retries: int = 3,
         timeout: int = 3600,
+        on_failure: TaskFailurePolicy = TaskFailurePolicy.ABORT_WORKFLOW,
```

Comment:
Lets just use a literal here as we do in the rest of the SDK implementation
```suggestion
        on_failure: Literal["abortWorkflow", "skipTask"] = "abortWorkflow",
```